readme.md

```diff
@@ -602,6 +602,7 @@
 - [Humane Technology](https://github.com/humanetech-community/awesome-humane-tech#readme) - Open source projects that help improve society.
 - [Speakers](https://github.com/karlhorky/awesome-speakers#readme) - Conference and meetup speakers in the programming and design community.
 - [Software Patreons](https://github.com/uraimo/awesome-software-patreons#readme) - Fund individual programmers or the development of open source projects.
+- [Bitcoin Payment Processors](https://github.com/alexk111/awesome-bitcoin-payment-processors#readme) - Open source and SaaS to accept Bitcoin payments.
```

Comment:
I think the description here could be better written.

Reply:
Maybe just `Accept Bitcoin payments.`?

Reply:
Updated